systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesIsolatedST.java

```diff
@@ -330,4 +333,72 @@ void testSwitchingStrimziPodSetFeatureGateOnAndOff(ExtensionContext extensionCon
 
         ClientUtils.waitForClientsSuccess(producerName, consumerName, INFRA_NAMESPACE, messageCount);
     }
+
+    /**
+     * UseKRaft feature gate
+     */
+    @IsolatedTest("Feature Gates test for enabled UseKRaft gate")
+    @Tag(INTERNAL_CLIENTS_USED)
+    public void testKRaftMode(ExtensionContext extensionContext) {
+        assumeFalse(Environment.isOlmInstall() || Environment.isHelmInstall());
+
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        final String producerName = "producer-test-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String consumerName = "consumer-test-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String topicName = KafkaTopicUtils.generateRandomNameOfTopic();
+
+        final LabelSelector zkSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.zookeeperStatefulSetName(clusterName));
+        final LabelSelector kafkaSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.kafkaStatefulSetName(clusterName));
+
+        int messageCount = 180;
```

Comment:
This can be used from `AbstractST` -> `MESSAGE_COUNT`

Reply:
But that has only 100 messages. 180 fits this test better.

Reply:
Ah sorry, I missed that there is 180. My bad :)